Uchu.StandardScripts/NinjagoMonastery/SkeletonTowers.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Uchu.Core;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.NinjagoMonastery
+{
+    [ZoneSpecific(2000)]
+    public class SkeletonTowers : NativeScript
```

Comment:
Can this be an `ObjectScript` instead of a `NativeScript`?

Reply:
In theory all scripts could be object scripts

Reply:
Made this one an object script, there are a few non-object scripts left (e.g. LearnSpinjitzu and SkeletonBuffs) but I wasn't able to find a good way to convert those (the implementation in live was possibly not using a script)